### PR TITLE
initial state cell editing

### PR DESCRIPTION
**Pull request by tishko0**

### CODE REVIEW

File: doc/en/components/grids/_shared/cell-editing.md [hunk 1]
```diff
@@ -323,6 +323,69 @@ Working sample of the above can be found here for further referencee:
 
 <!-- end: WebComponents -->
 
+<!-- React -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. First create the column as you usually would:
+
+```html
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/cell-editing.md:330]: This should be a tsx code snippet

File: doc/en/components/grids/_shared/cell-editing.md [hunk 2]
```diff
@@ -323,6 +323,69 @@ Working sample of the above can be found here for further referencee:
...
+    id="column1">
+</igrColumn>
+
+```
+
+and pass the templates to this column in the index.ts file:
+
+```ts
+
+constructor(props: any) {
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/cell-editing.md:347]: rework to not use Classes

File: doc/en/components/grids/_shared/cell-editing.md [hunk 3]
```diff
@@ -323,6 +323,69 @@ Working sample of the above can be found here for further referencee:
...
+        let uniqueValues: any = [];
+        for(const i of (this.webGridCellEditSampleRoleplay as any)){
+            const field: string = ctx.cell.column.field;
+            if(uniqueValues.indexOf(i[field]) === -1 )
+            {
+                cellValues.push(html`<igc-select-item value=${i[field]}>${(i[field])}</igc-select-item>`);
+                uniqueValues.push(i[field]);
+            }
+        }
+        return html`
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/cell-editing.md:375]: This is a WC template, not a react one

File: doc/en/components/grids/_shared/cell-editing.md [hunk 4]
```diff
@@ -751,6 +814,10 @@ In case you would like to change some of the colors, you need to set a class for
 <IgbGrid Class="grid"></IgbGrid>
 ```
 
+```tsx
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/cell-editing.md:817]: You can switch the WC codesnipet to ```html because right now it shows for React

**Comment by ChronosSF** [comment]
@tishko0 or @IMinchev64 , could you please just copy-paste the cell editing content in another branch that's properly based on react-documentation. This PR contains additional files that have separate PRs so it'll be difficult to track changes and review between them.

**Comment by IMinchev64** [comment]
> @tishko0 or @IMinchev64 , could you please just copy-paste the cell editing content in another branch that's properly based on react-documentation. This PR contains additional files that have separate PRs so it'll be difficult to track changes and review between them.

New PR with just the cell-editing topic: https://github.com/IgniteUI/igniteui-xplat-docs/pull/986

**Pull request closed without merging**